src/FsAutoComplete/CodeFixes/AddMissingXmlDocumentation.fs

```diff
@@ -0,0 +1,200 @@
+module FsAutoComplete.CodeFix.AddMissingXmlDocumentation
+
+open FsAutoComplete
+open FsAutoComplete.CodeFix.Types
+open FsAutoComplete.LspHelpers
+open FSharp.Compiler.Syntax
+open FSharp.Compiler.Text.Range
+open FSharp.Compiler.Xml
+open FsToolkit.ErrorHandling
+open Ionide.LanguageServerProtocol.Types
+open System
+
+let title = "Add missing XML documentation"
+
+let private tryGetExistingXmlDoc (pos: FSharp.Compiler.Text.Position) (xmlDoc: PreXmlDoc) =
+  let containsPosAndSummaryPresent (xd: PreXmlDoc) =
+    let d = xd.ToXmlDoc(false, None)
```

Comment:
I was going to ask about doing this work, but then I remembered I raised https://github.com/dotnet/fsharp/issues/14882 already to track it :)